fixtures/wrangler-dev-api-app/src/wrangler-dev.mjs

```diff
@@ -0,0 +1,12 @@
+import undici from "undici";
+import wrangler from "wrangler";
+
+//since the script is invoked from the directory above, need to specify index.js is in src/
+const worker = await wrangler.unstable_dev("src/index.js");
+
+const resp = await undici.fetch("http://localhost:8787/");
```

Comment:
It might be nice to hide the undici library, and also the underlying port, side of this and provide:

```ts
worker.fetch()
```

This method could take a path (or even a full URL if you wanted to simulate a custom route). This would also mitigate the case where port 8787 was not available and we ended up jumping on a different port, which I think we can do?